support-payment-api/src/main/scala/backend/SubscribeWithGoogleBackend.scala

```diff
@@ -64,8 +64,8 @@ case class SubscribeWithGoogleBackend(databaseService: ContributionsStoreService
       _ => {
         logger.warn(s"unable to get identity id for email ${googleRecordPayment.email} with paymentId : ${googleRecordPayment.paymentId}, tracking acquisition anyway")
         handleContributionWithoutIdentity(googleRecordPayment, clientBrowserInfo).value
-      }, idWithGuestToken =>
```

Comment:
This backend could be deleted now couldn't it? We don't support Subscribe with Google any more

Reply:
yes! I've been meaning to do this for about a year now